tests/org.flatpak.Flatpak.yml

```diff
@@ -21,3 +21,24 @@ modules:
           type: anitya
           project-id: 6377
           url-template: https://github.com/flatpak/flatpak/releases/download/$version/flatpak-$version.tar.xz
+          
+  - name: boost
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2
+        sha256: "0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Either set this to actual checksum here or to all zeros in the test assertion, otherwise the assertion always succeeds. Same goes for ghostscript.